apollo-common/src/main/resources/application.properties

```diff
@@ -20,3 +20,6 @@ logging.file.max-size=50MB
 logging.file.max-history=10
 
 management.endpoints.web.exposure.include=info,health,metrics,prometheus
+
+# Project information
+spring.info.git.location=classpath:apollo.git.properties
```

Comment:
Why not name it `apollo-git.properties`? I think it will make the filename suffix more clearly.

Reply:
This is a good point. Just updated as suggested.